.github/workflows/ci.yml

```diff
@@ -231,6 +231,9 @@ jobs:
       - run: cypress/wait-for-frontend-to-start.sh
         working-directory: app
 
+      - run: docker compose up -d mail-function mail
```

Comment:
I would leave that. (Else it builds the container every time and this may fail the test)